Documentation/ApiOverview/ExtensionScanner/Index.rst

```diff
@@ -1,34 +1,34 @@
 .. include:: /Includes.rst.txt
-
+.. index:: Extension Scanner
```

Comment:
But this is `Extension scanner` in the following headline. I guess that's what should be considered right?!